WordPress/src/androidTest/java/org/wordpress/android/e2e/EditorTests.java

```diff
@@ -1,29 +1,69 @@
 package org.wordpress.android.e2e;
 
+import android.Manifest.permission;
 import android.support.test.rule.ActivityTestRule;
+import android.support.test.rule.GrantPermissionRule;
 
 import org.junit.Rule;
 import org.junit.Test;
+import org.wordpress.android.R;
 import org.wordpress.android.e2e.components.MasterbarComponent;
 import org.wordpress.android.e2e.pages.EditorPage;
 import org.wordpress.android.e2e.pages.MySitesPage;
 import org.wordpress.android.support.BaseTest;
 import org.wordpress.android.ui.WPLaunchActivity;
 
+import java.time.Instant;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.Espresso.pressBack;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
 import static junit.framework.TestCase.assertTrue;
 import static org.wordpress.android.BuildConfig.E2E_WP_COM_USER_SITE_ADDRESS;
+import static org.wordpress.android.support.WPSupportUtils.checkViewHasText;
 import static org.wordpress.android.support.WPSupportUtils.sleep;
+import static org.wordpress.android.support.WPSupportUtils.waitForElementToNotBeDisplayed;
 
 public class EditorTests extends BaseTest {
     @Rule
     public ActivityTestRule<WPLaunchActivity> mActivityTestRule = new ActivityTestRule<>(WPLaunchActivity.class);
 
+    @Rule
+    public GrantPermissionRule mRuntimeImageAccessRule = GrantPermissionRule.grant(permission.READ_EXTERNAL_STORAGE);
```

Comment:
This should resolve the permissions issue with the media picker in the test:

```suggestion
    public GrantPermissionRule mRuntimeImageAccessRule = GrantPermissionRule.grant(permission. WRITE_EXTERNAL_STORAGE);
```

Reply:
Ahhh 🤦🏽‍♂️ Thanks for catching this!